system/Security/CheckPhpIni.php

```diff
@@ -134,12 +135,28 @@ public static function checkIni(): array
             'date.timezone'           => ['recommended' => 'UTC'],
             'mbstring.language'       => ['recommended' => 'neutral'],
             'opcache.enable'          => ['recommended' => '1'],
-            'opcache.enable_cli'      => [],
-            'opcache.jit'             => [],
-            'opcache.jit_buffer_size' => [],
+            'opcache.enable_cli'      => ['recommended' => '1'],
+            'opcache.jit'             => ['recommended' => 'tracing'],
+            'opcache.jit_buffer_size' => ['recommended' => '256', 'remark' => 'Adjust with your free space of memory'],
             'zend.assertions'         => ['recommended' => '-1'],
         ];
 
+        if ($argument === 'opcache') {
+            $items = [
+                'opcache.enable'                  => ['recommended' => '1'],
+                'opcache.enable_cli'              => ['recommended' => '1'],
```

Comment:
Why `opcache.enable_cli` is `1`? 

In my understanding, it does not improve performance unless you set `opcache.file_cache` correctly.
https://stackoverflow.com/questions/77778860/proper-opcache-configuration-for-cli